wsagent/che-core-api-debug-shared/src/main/java/org/eclipse/che/api/debug/shared/model/DebugSessionState.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Rogue Wave Software, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Rogue Wave Software, Inc. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.debug.shared.model;
+
+/**
+ * Debug session state
+ * 
+ * @author Bartlomiej Laczkowski
+ */
+public interface DebugSessionState {
+    
+    /**
+     * Returns debug session state.

```

Comment:
it's because it's on "String getDebuggerType();"
so text should be IMHO returns debug session type